pallets/subtensor/Cargo.toml

```diff
@@ -37,8 +37,8 @@ ndarray = { version = "0.15.0", default-features = false }
 hex = { version = "0.4", default-features = false }
 
 # Used for sudo decentralization
-pallet-collective = { version = "4.0.0-dev", default-features = false, path = "../collective" }
-pallet-membership = {version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.39" }
+#pallet-collective = { version = "4.0.0-dev", default-features = false, path = "../collective" }
```

Comment:
Instead of commenting these out, these need to be removed entirely. Git history will maintain all of this